### DISCUSSION TITLE
Use unplugin-vue-components with quasar vite cli

### DISCUSSION THREAD

**Discussion by lukadriel7** [quasarframework/quasar#13260] [General - Components / Directives / etc]
Hello everyone, I just started exploring the vite powered quasar app today. I installed [@vueuse/components](https://github.com/vueuse/vueuse) and [unplugin-vue-components](https://github.com/antfu/unplugin-vue-components) but I can't seem to make the vueuse resolver work. In the options of the documentation of unplugin-vue-components, the resolvers are functions. I don't know if Quasar supports those options in the config file. Has anyone made it work ?

**Comment by jclaveau** [comment]
Here is an example: https://stackoverflow.com/questions/75838485/is-there-any-way-to-use-unplugin-vue-components-with-quasar-cli-vite-version
